src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -185,7 +191,10 @@ export function DownloadOptionPanel (props) {
     };
 
     const dlTitle = get(dlParams, 'TitlePrefix', 'Download') + '-' + dlTitleIdx;
-
+    // IRSA-3944: SOFIA required a message to guide the user before downloading AOR such as:
+    // 'All levels associated with this AOR will be downloaded. To download individual data products, go to the relevant instrument tab.'
+    // PreTitleMessage can be added to the dlParams from the parent and passed here (FIREFLY-723)
+    const preTitleMessage = get(dlParams, 'PreTitleMessage', '')
```

Comment:
we don't need `get` anymore
```
const preTitleMessage = dlParams?.PreTitleMessage ?? '';
```

Reply:
cool, i'll change it before merging.

Reply:
@robyww are you sure? It says unresolved variable - are you sure of the syntax?

Reply:
yes, use it all the time

Reply:
Ok, i'll copy paste and merge.

Reply:
Yep, confirmed. Seems intellij is not good enough to interpret the syntax.